projects/gui/src/mainwindow.cpp

```diff
@@ -341,9 +341,9 @@ void MainWindow::setCurrentGame(const TabData& gameData)
 	if (gameData.game == m_game && m_game != 0)
 		return;
 
-	for (int i = 0; i < 2; i++)
+	for (auto & m_player : m_players)
```

Comment:
The **m_** prefix is used for member variables in classes. Since we're iterating over **m_players** the variable name should be **player**.